package.json

```diff
@@ -39,5 +39,10 @@
     "web-ext": "^2.4.0",
     "webpack": "^3.10.0"
   },
-  "license": "MPL-2.0"
+  "license": "MPL-2.0",
+  "main": "index.js",
```

Comment:
Can we remove these 3 keys (`main`, `directories`, `dependencies`)? They're not needed.

Reply:
In fact, the whole 8f2b7f22f0db2aa938542b254d9a458bab3bf518 commit can just be removed.